generate_stats.py

```diff
@@ -17,10 +17,9 @@ def report_stats(entity_ranges):
     ------
     entity_ranges : a dictionary with one entry for each entity type that is a Range of handles to that type
     """
-
     for entity_type, eh_range in entity_ranges.items():
         print("There are {} entities of type {}.".format(eh_range.size(),entity_type))
-
+    print('Type 0: Vertices \nType 2: Triangles \nType 11: EntitySets')
```

Comment:
This is a useful legend, but might be helpful to  implement a way to provide this information instead of the entity number in the previous print statement.

Reply:
Okay, I'll implement that in